tests/tls_e2e/client_enc/client.cpp

```diff
@@ -0,0 +1,433 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/edger8r/enclave.h>
+#include <openenclave/enclave.h>
+//#define OE_NEED_STDC_NAMES
+#include <openenclave/corelibc/sys/socket.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/report.h>
+#include <openenclave/internal/tests.h>
+
+#include <openenclave/corelibc/arpa/inet.h>
+#include <openenclave/corelibc/netdb.h>
+#include <openenclave/corelibc/netinet/in.h>
+#include <openenclave/internal/posix/device.h>
+
+//#include <bits/stdfile.h> // For stderr & FILE
+#include <errno.h> // For errno & error defs
```

Comment:
Cleanup the commented out stuff if you don't need them

Reply:
Removed